app/src/main/java/com/gathering/android/auth/signin/repo/ApiSignInRepository.kt

```diff
@@ -22,10 +24,17 @@ class ApiSignInRepository @Inject constructor(
                 response: Response<AuthorizedResponse>
             ) {
                 if (response.isSuccessful) {
```

Comment:
You can use early return here, as a rule of thumb we try to return early if there is an exception case instead of if/else.

For example:
Instead of ->
```
if (response.successful) {
 // Do if instructions
} else {
 // Do else instructions  
}
```
Do this ->

```
if (!response.successful) {
 // Do else instructions
return
}

// Do if instructions
```